openedx/adg/common/custom_registration_form/apps.py

```diff
@@ -0,0 +1,10 @@
+"""
+Application for custom registration form
+"""
+
+
```

Comment:
Let's please remove these extra spaces from all files, and also please rebase from master because we've integrated github actions for linting now.